repositories/user.repository.js

```diff
@@ -0,0 +1,31 @@
+var mongoose = require('mongoose');
+
+var UserSchema = new mongoose.Schema({
+    username: {
+        type: String,
+        required: true
+    },
+    password: {
+        type: String,
+        required: true
+    },
+    firstname: {
+        type: String,
+        required: true
+    },
+    lastname: {
+        type: String,
+        required: true
+    },
+    email: {
+        type: String,
+        required: true
+    },
+    contact: {
+        type: Number,
+        required: true
```

Comment:
Consider making this field optional and not required.

Reply:
done